### PR TITLE
Iabthao

### PR DESCRIPTION
**Pull request by iabthao**
Here is the rough draft of the vehicle class following the diagram. It's a very light class at the moment, if I'm missing anything, let me know, and if you need help with other part of code, let me know as well.

### CODE REVIEW

**Comment by CwongM** [comment]
Looks good to me! It would be nice to have an overriden toString method to show the attributes of a Vehicle object. I'm having the Dealership class get this information to return a string for the console to print out.

**Comment by iabthao** [comment]
I added  the toString method, will be adjusted later when we have a visual/driver.

File: .project [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<projectDescription>
+	<name>ICS372-Assignment1</name>
+	<comment></comment>
+	<projects>
+	</projects>
+	<buildSpec>
+	</buildSpec>
+	<natures>
```
Comment (tomatogrove) [inline .project:9]: The Vehicle file looks good. Just get rid of the .project file and I'll approve it

Comment (iabthao) [reply .project:9]: resolved, please push in main, thanks!

**Pull request merged**